packages/builder/src/components/common/LinkedRowSelector.svelte

```diff
@@ -41,13 +41,22 @@
     table.
   </Label>
 {:else}
-  <Multiselect
-    secondary
-    bind:value={linkedRows}
-    {label}
-    placeholder="Choose some options">
-    {#each rows as row}
-      <option value={row._id}>{getPrettyName(row)}</option>
-    {/each}
-  </Multiselect>
+  {#if schema.relationshipType === 'one-to-many'}
+    <Select secondary on:change={e => linkedRows = [e.target.value]} name={label} {label}>
```

Comment:
We should add the `thin` prop here to keep it visually consistent with the other components too.

Reply:
Makes sense.